Tools/scripts/run_coverage.py

```diff
@@ -148,6 +153,7 @@ def run_full(self, use_example=False):
         self.progress("Running tests")
         SPEEDUP = 5
         TIMEOUT = 14400
+        os.environ["COVERAGE"] = "1"
```

Comment:
Do we still need this?

Should probably be `AP_COVERAGE`, BTW.

Reply:
removed !